package.json

```diff
@@ -497,6 +497,12 @@
         "title": "Hide Security Hotspot",
         "category": "SonarLint",
         "enablement": "view == SonarLint.SecurityHotspots && viewItem == remoteHotspotItem"
+      },
+      {
+        "command": "SonarLint.ChangeHotspotStatus",
+        "title": "Set status",
```

Comment:
I think `Review Hotspot` or `Change status` or `Set review output` would be better

Reply:
Addressed in [7c93c61](https://github.com/SonarSource/sonarlint-vscode/pull/411/commits/7c93c6130d509717de007ea8ec719bd89279f085)